README.md

```diff
@@ -55,6 +56,7 @@ systemd-python
 
 Once dependencies are installed, the following scripts can be run to build and test the project:
 ```
+./deps.sh
```

Comment:
This command is a little at odds with the text above, maybe separate paragraph about deps (one time thing) from the normal build steps?

Reply:
That's a good point, I broke this out separately and also provided `git submodule` instructions.